docs/learn/dapp-developer/register-lsp12-issued-assets.md

```diff
@@ -0,0 +1,204 @@
+
+The following code snippets require the installation of the following libraries:
+
+- [`ethers.js`](https://github.com/ethers-io/ethers.js/) or [`web3.js`](https://www.npmjs.com/package/web3)
+- [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
+- [`@erc725/erc725.js`](https://github.com/ERC725Alliance/erc725.js/)
+
+<Tabs groupId="web3-lib">
+  <TabItem value="web3js" label="web3.js">
+
+```shell
+npm install web3 @lukso/lsp-smart-contracts @erc725/erc725.js
+```
+
+  </TabItem>
+  <TabItem value="ethersjs" label="ethers.js">
+
+```shell
+npm install ethers @lukso/lsp-smart-contracts @erc725/erc725.js
+```
+
+  </TabItem>
+</Tabs>
+
+## Imports and constants
+
+Import `web3.js`/`ethers`, the [`LSP8IdentifiableDigitalAsset`](../../contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8IdentifiableDigitalAsset.md) ABI from [`@lukso/lsp-smart-contracts`](../../contracts/introduction.md) and create an instance of this contract with the `lsp8ContractAddress`.
+
+<Tabs groupId="web3-lib">
+  <TabItem value="web3js" label="web3.js">
+
+```javascript
+import Web3 from 'web3';
+import { ERC725 } from '@erc725/erc725.js';
+import LSP12Schema from '@erc725/erc725.js/schemas/LSP12IssuedAssets.json';
+
+import UniversalProfileArtifact from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
+
+// We will register the issued assets by setting the following LSP12 data keys
+// - LSP12IssuedAssets[]
+// - LSP12IssuedAssetsMap:<asset-address>
+
+// add the type of asset (LSP7 or LSP8) and their address in the object list below
+const issuedAssets = [
+  {
+    interfaceId: INTERFACE_IDS.LSP7DigitalAsset,
+    address: '0xf7056bdE90f494F55967858F1e9E4AFB1026C5C8',
+  },
+  {
+    interfaceId: INTERFACE_IDS.LSP8IdentifiableDigitalAsset,
+    address: '0xf651b88925C0B6C81Ad6f658a2F104226d837F60',
+  },
+  //   {
+  //     interfaceId: LSP7 or LSP8 interface ID (or other),
+  //     address: '0xasset-address',
+  //   },
+];
+
+const web3 = new Web3(window.lukso);
+await web3.eth.requestAccounts();
+const accounts = await web3.eth.getAccounts();
+```
+
+  </TabItem>
+  <TabItem value="ethersjs" label="ethers.js">
+
+```javascript
+import { ethers } from 'ethers';
+
+import { ERC725 } from '@erc725/erc725.js';
+import LSP12Schema from '@erc725/erc725.js/schemas/LSP12IssuedAssets.json';
+
+import UniversalProfileArtifact from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
+
+// We will register the issued assets by setting the following LSP12 data keys
+// - LSP12IssuedAssets[]
+// - LSP12IssuedAssetsMap:<asset-address>
+
+// add the type of asset (LSP7 or LSP8) and their address in the object list below
+const issuedAssets = [
```

Comment:
In the guide, you set the issued assets for the first time, but we should explain/include how to add more assets- in case the issued assets and issued assets map were not empty. Meaning people would have to get the original content, decode it, and add the new asset to the list, right?

This should be done in both examples.

Reply:
> In the guide, you set the issued assets for the first time, but we should explain/include how to add more assets- in case the issued assets and issued assets map were not empty. Meaning people would have to get the original content, decode it, and add the new asset to the list, right?
> 
> This should be done in both examples.


erc725.js does not enable this now. It sets only from scratch. We can add a callout warning at the top, and update the guides once we have this feature